test/transformers/test_active_space_transformer.py

```diff
@@ -0,0 +1,150 @@
+                                            [[0.0, 0.0, 0.03600701],
+                                             [0.0, 0.0, 0.028244],
+                                             [0.03600701, 0.028244, 0.0]]],
+                                           [[[-0.05593597, 0.01937529, 0.0],
+                                             [0.01937529, 0.02020237, 0.0],
+                                             [0.0, 0.0, 0.01405676]],
+                                            [[0.30428426, 0.02020237, 0.0],
+                                             [0.02020237, 0.48162669, 0.0],
+                                             [0.0, 0.0, 0.40269913]],
+                                            [[0.0, 0.0, 0.028244],
+                                             [0.0, 0.0, 0.0564951],
+                                             [0.028244, 0.0564951, 0.0]]],
+                                           [[[0.0, 0.0, 0.03600701],
+                                             [0.0, 0.0, 0.028244],
+                                             [0.03600701, 0.028244, 0.0]],
+                                            [[0.0, 0.0, 0.028244],
+                                             [0.0, 0.0, 0.0564951],
+                                             [0.028244, 0.0564951, 0.0]],
+                                            [[0.36650821, 0.01405676, 0.0],
+                                             [0.01405676, 0.40269913, 0.0],
+                                             [0.0, 0.0, 0.44985904]]]])
+
+        assert np.allclose(q_molecule_reduced.mo_onee_ints, expected_mo_onee_ints)
+        assert np.allclose(q_molecule_reduced.mo_eri_ints, expected_mo_eri_ints)
+        assert np.isclose(q_molecule_reduced.energy_shift['inactive_energy'], -14.2538029231)
+
+    def test_arbitrary_active_orbitals(self):
+        """Test manual selection of active orbital indices."""
+        driver = PySCFDriver(atom='H 0 0 0; H 0 0 0.735', basis='6-31g')
+        q_molecule = driver.run()
+
+        trafo = ActiveSpaceTransformer(num_electrons=2, num_orbitals=2, active_orbitals=[0, 2])
+        q_molecule_reduced = trafo.transform(q_molecule)
+
+        expected_mo_onee_ints = np.asarray([[-1.24943841, -0.16790838], [-0.16790838, -0.18307469]])
+        expected_mo_eri_ints = np.asarray([[[[0.65209847, 0.16790822], [0.16790822, 0.53250905]],
+                                            [[0.16790822, 0.10962908], [0.10962908, 0.11981429]]],
+                                           [[[0.16790822, 0.10962908], [0.10962908, 0.11981429]],
+                                            [[0.53250905, 0.11981429], [0.11981429, 0.46345617]]]])
+
+        assert np.allclose(q_molecule_reduced.mo_onee_ints, expected_mo_onee_ints)
+        assert np.allclose(q_molecule_reduced.mo_eri_ints, expected_mo_eri_ints)
+        assert np.isclose(q_molecule_reduced.energy_shift['inactive_energy'], 0.0)
+
+    def test_error_raising(self):
```

Comment:
Can we either use subTest here or better yet use ddt to supply parameters since its the same test with different parameterization.